packages/sources/synthetix-debt-pool/src/endpoint/debt.ts

```diff
@@ -72,6 +71,30 @@ const DEBT_POOL_ABI = [
     stateMutability: 'view',
     type: 'function',
   },
+  {
+    constant: true,
+    inputs: [],
+    name: 'totalNonSnxBackedDebt',
+    outputs: [
+      { internalType: 'uint256', name: 'excludedDebt', type: 'uint256' },
+      { internalType: 'bool', name: 'isInvalid', type: 'bool' },
+    ],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function',
+  },
+]
```

Comment:
Might be nice to include the source for these ABIs as well

Reply:
what's the source?

Reply:
I was thinking either the address of the deployed instance of the contract so people can check it on etherscan, or if synthetix has reference documentation somewhere...